internal/ingress/annotations/authtls/main.go

```diff
@@ -31,10 +33,15 @@ import (
 const (
 	defaultAuthTLSDepth     = 1
 	defaultAuthVerifyClient = "on"
+	defaultOCSP             = "off"
+	defaultOCSPCache        = "off"
 )
 
 var (
 	authVerifyClientRegex = regexp.MustCompile(`on|off|optional|optional_no_ca`)
+	authOCSPRegex         = regexp.MustCompile(`on|off|leaf`)
```

Comment:
What anout ^(?on|off|leaf)$

Reply:
Yeah, that would work too :) For this regex I kept the same regex structure as the `authVerifyClientRegex`. For the `has to start with http` one I opted for the `^` anchor because the requirement there is that it has to start with `http://`. 

I don't feel super strong about this, but do enjoy consistency in files and readability with regex is always a thing.